Build/libHttpClient.141.Android.Java/app/src/main/java/com/xbox/httpClient/HttpClientResponse.java

```diff
@@ -0,0 +1,61 @@
+
+package com.xbox.httpclient;
+
+import android.util.Log;
+
+import java.io.IOException;
+import okhttp3.Response;
+
+class HttpClientResponse
+{
+    private Response okHttpResponse;
+
+    public HttpClientResponse(Response sourceResponse) {
+        this.okHttpResponse = sourceResponse;
+    }
+
+    public boolean succeeded() {
+        return this.okHttpResponse.isSuccessful();
+    }
+
+    public int getNumHeaders() {
+        return this.okHttpResponse.headers().size();
+    }
+
+    public String getHeaderNameAtIndex(int index) {
+        if (index >= 0 && index < this.okHttpResponse.headers().size()) {
+            return this.okHttpResponse.headers().name(index);
+        } else {
+            return null;
+        }
+    }
+
+    public String getHeaderValueAtIndex(int index) {
+        if (index >= 0 && index < this.okHttpResponse.headers().size()) {
+            return this.okHttpResponse.headers().value(index);
+        } else {
+            return null;
+        }
+    }
+
+    public String getResponseString() {
+        try {
+            return this.okHttpResponse.body().string();
+        } catch (IOException e) {
+            return null;
+        }
+    }
```

Comment:
is this used anywhere?

Reply:
Nope, I've removed it.

---
In reply to: [189760666](https://github.com/Microsoft/libHttpClient/pull/165#discussion_r189760666) [](ancestors = 189760666)